test/unit/unit-test1.spec.js

```diff
@@ -0,0 +1,174 @@
+
```

Comment:
Change file name to something more meaningful. Eg. `secres-scnner.spec.js` or something more appropriate.

Reply:
Changed the file name to secrets-scanner.spec.js